.github/workflows/tii-micrortps-agent.yaml

```diff
@@ -0,0 +1,47 @@
+name: tii-micrortps-agent
+
+on:
+  push:
+    branches: master
+    tags:
+      - 'v*'
+      - v[0-9]+.[0-9]+.[0-9]+
+      - v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tii-micrortps-agent:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: docker/setup-buildx-action@v1
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/tiiuae/tii-micrortps-agent
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=sha
+            type=raw,value=latest
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        if: github.event_name == 'push'
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GCR_PACKAGES_USER }}
+          password: ${{ secrets.GCR_PACKAGES_TOKEN }}
+
+      - name: Build tii-micrortps-agent image and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile.containerize
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Can you add the labels tag to have container metadata:
`labels: ${{ steps.meta.outputs.labels }}`